criu/uffd.c

```diff
@@ -270,6 +270,10 @@ int uffd_open(int flags, unsigned long *features)
 	uffd = syscall(SYS_userfaultfd, flags);
 	if (uffd == -1) {
 		pr_info("Lazy pages are not available: %s\n", strerror(errno));
+		if (errno == EPERM) {
+			pr_info("Uffd loader is not loaded \n");
+			return 0;
+		}
```

Comment:
I worry that this patch introduces a bug. E.g. If we got EPERM errno from uffd_open() and return 0 from uffd_open() to setup_uffd(). Next send_uffd would send random fd at index "0" into lazy pages socket, which is probably unexpected.